crates/orderbook/openapi.yml

```diff
@@ -965,3 +989,18 @@ components:
             the fee after this expiration date. Encoded as ISO 8601 UTC.
           type: string
           example: "1985-03-10T18:35:18.814523Z"
+    SolverCompetitionResponse:
+      type: object
+      description: Maps from solver name to object describing that solver's settlement.
```

Comment:
I don't think it can be a map since solvers can technically propose multiple solutions (although maybe we don't want to include those...)? Maybe it should be an array instead.